tests/FINDOLOGIC/Tests/XmlSerializationTest.php

```diff
@@ -187,4 +195,28 @@ public function testUsergroupVisibilitiesAreExported()
 
         $this->assertPageIsValid($page);
     }
+
+    /**
+     * Generates minimal XML-items
+     *
+     * @param $numberOfItems The number of items to generate.
+     * @return array The generated items.
+     */
+    public function itemGenerator($numberOfItems)
```

Comment:
Currently only used in one place. Are there more where it could be useful?

Reply:
Moved item generation directly to the corresponding test. If there are any other tests needed with more than one item we should add a method like this.